src/main/scala/cromwell/engine/backend/ExecutionResult.scala

```diff
@@ -21,8 +21,13 @@ final case class SuccessfulBackendCallExecution(outputs: CallOutputs, executionE
  */
 case object AbortedExecution extends ExecutionResult
 
-/**
- * Failed execution, possibly having a return code.
- */
-final case class FailedExecution(e: Throwable, returnCode: Option[Int] = None) extends ExecutionResult
+sealed trait FailedExecution extends ExecutionResult {
+  def e: Throwable
```

Comment:
I'll admit that when I wrote this I was imagining using `ErrorOr[A]` but there's not actually an `Or` here :) 

The real culprit that I'm complaining about is Try because then you have to unbox the Try and the exception. Considering how often I moan about stringly typed systems and how all strings should be wrapped by something I suppose an exception is good enough for things like this..

Disregard me.